contracts/StorageContract.sol

```diff
@@ -379,6 +401,14 @@ abstract contract StorageContract is DecentralizedKV, ReentrancyGuardTransient {
         _rewardMiner(_shardId, _miner, mineTs, diff);
     }
 
+    /// @notice Withdraw treasury fund
+    function withdraw(uint256 _amount) public {
+        require(accPrepaidAmount >= prepaidAmount + _amount, "StorageContract: not enough prepaid amount");
```

Comment:
is there any limitation on who can withdraw?

Reply:
It's ok for anyone to do this.